### PR TITLE
Release/1.0.6

### PR DESCRIPTION
**Pull request by emmettculley**
Cause upgrade to php 7.3.x

### CODE REVIEW

**Review by chessracer — APPROVED** [review]

File: Dockerfile [hunk 1]
```diff
@@ -31,19 +31,20 @@ RUN apk --update --no-cache add wget \
   php7-xmlreader \
   php7-xmlwriter \
   php7-zip \
-  php7-zlib
+  php7-zlib \
+  composer
 
-RUN apk add --no-cache --repository http://dl-3.alpinelinux.org/alpine/edge/testing gnu-libiconv
+#RUN apk add --no-cache --repository http://dl-cdn.alpinelinux.org/alpine/edge/community gnu-libiconv
```
Comment (chessracer) [inline Dockerfile]: In future, I'd suggest removing unused code rather than commenting it out.

**Review by chessracer — CHANGES_REQUESTED** [review]
out-of-the-box, this image doesn't run: 

`03:19:50 ~$ docker run --rm ccctechcenter/composer-laravel:1.0.6
Unable to find image 'ccctechcenter/composer-laravel:1.0.6' locally
1.0.6: Pulling from ccctechcenter/composer-laravel
cbdbe7a5bc2a: Already exists 
eaeaad05d9ca: Pull complete 
b9a056d91223: Pull complete 
Digest: sha256:005036c4d8de8059af1e91899c1a62db087c4f6275038df9de3e47b899cd24d7
Status: Downloaded newer image for ccctechcenter/composer-laravel:1.0.6
Usage:
  help [options] [--] [<command_name>]

Arguments:
  command                        The command to execute
  command_name                   The command name [default: "help"]

Options:
      --xml                      To output help as XML
      --format=FORMAT            The output format (txt, xml, json, or md) [default: "txt"]
      --raw                      To output raw command help
  -h, --help                     Display this help message
  -q, --quiet                    Do not output any message
  -V, --version                  Display this application version
      --ansi                     Force ANSI output
      --no-ansi                  Disable ANSI output
  -n, --no-interaction           Do not ask any interactive question
      --profile                  Display timing and memory usage information
      --no-plugins               Whether to disable plugins.
  -d, --working-dir=WORKING-DIR  If specified, use the given directory as working directory.
      --no-cache                 Prevent use of the cache
  -v|vv|vvv, --verbose           Increase the verbosity of messages: 1 for normal output, 2 for more verbose output and 3 for debug

Help:
  The help command displays help for a given command:
  
    php /usr/bin/composer help list
  
  You can also output the help in other formats by using the --format option:
  
    php /usr/bin/composer help --format=xml list
  
  To display the list of available commands, please use the list command.
`

**Comment by emmettculley** [comment]
That seems to be the correct output from running 

composer --help

 Plus composer does run correctly in the build/deployment of laravel-cid containers.

**Comment by emmettculley** [comment]
Redid to use --version instead of --help for entry point parameter.   Now build result is pretier and still composer runs upon deployment.

**Pull request merged**